osprey/processes/wps_convert.py

```diff
@@ -24,13 +24,38 @@ def __init__(self):
             "complete": 100,
         }
         inputs = [
-            LiteralInput(
-                "convert_config",
-                "Configuration",
-                abstract="Path to input configuration file or input dictionary",
-                data_type="string",
-            ),
             log_level,
+            ComplexInput(
+                "uhs_files",
+                "UHS_Files",
+                abstract="Path to UHS file (required)",
+                min_occurs=1,
+                supported_formats=[FORMATS.TEXT],
+            ),
+            ComplexInput(
+                "station_file",
+                "Station_FILE",
+                abstract="Path to stations file (required)",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.TEXT],
+            ),
+            ComplexInput(
+                "domain",
+                "Domain",
+                abstract="Path to CESM complaint domain file (required)",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+            ),
+            ComplexInput(
+                "config_file",
+                "Convert Configuration",
+                abstract="Path to input configuration file for Convert process (optional)",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[Format("text/cfg", extension=".cfg")],
```

Comment:
In `thunderbird`, yes we were. And it kind of happened in `osprey` as well. So, the strange behaviour was that `vars(request.inputs["updates_file"][0])._file` was shown as `None` when running in docker containers, and I assumed the copy of the input file, `input.extension`, is not being created in `tmp`. This bugged me for a long time. However, with curiosity, when I just tried logging `request.inputs["updates_file"][0].file` to see if it is `None` for sure, it gave a filepath with actual content in it.

In short, we still see some bizarre behaviour when logging out `__dict__` attribute of inputs with a non-predefined format, but we still can _"use"_ the `file` attribute. I will have to make sure if it applies to `thunderbird` as well, and it would be great if it works.